stack.yaml

```diff
@@ -3,3 +3,6 @@ resolver: lts-11.10
 flags:
   yaml:
     no-examples: false
+
+docker:
+  enable: true
```

Comment:
Please remove this, Docker should not be the default for the repo.

Reply:
Oops. Accidental, will do.